learn/wavemaker-release-notes/v11-5-0.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "WaveMaker 11.5.0 - Release date: 03 December 2023"
+id: "v11-5-0"
+sidebar_label: "v11.5.0 - latest"
+---
+
+WaveMaker has released version 11.5.0 with new features like Appium Support, Dockerfiles in a project, importing icon font family, and enhancements like Angular framework support for Prefab workspace, NG studio upgrade to version 15, changes in project shell web template and other bug fixes.
+
+## New Features
+
+### Dockerfiles in Project
+
+Introduced Dockerfiles in the project to build and deploy applications. This helps the user to use an appropriate Dockerfile for a specific project which stores all the prerequisite versions that are needed for the application.
```

Comment:
there should be a doc for this, on how to use this Dockerfile

Reply:
Includue the doc link which we have.